plugins/HHAnalyzer.cc

```diff
@@ -288,8 +292,9 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
     // ***** 
     // Adding MET(s)
     // ***** 
-    // FIXME: add back standard MET
-//    const METProducer& stdmet = producers.get<METProducer>("met");
+    // FIXME: add back standard MET 
+    // CAREFULL WHEN FIXING THE ABOVE : for now stdmet is added as llmetjj.pfmet, nohf_met is added as llmetjj.nohf_met 
+    const METProducer& stdmet = producers.get<METProducer>("met");
```

Comment:
As said above, not sure I like this, it won't be clear for the angles which met you are referring to...